ddspipe_yaml/src/cpp/YamlReader_types.cpp

```diff
@@ -428,6 +430,82 @@ void YamlReader::fill(
     }
 }
 
+/************************
+* LOGGING CONFIGURATION *
+************************/
+
+using VersosityKind = fastdds::dds::Log::Kind;
+
+template <>
+DDSPIPE_YAML_DllAPI
+void YamlReader::fill<LogFilter>(
+        LogFilter& object,
+        const Yaml& yml,
+        const YamlReaderVersion version)
+{
+    if (is_tag_present(yml, LOG_FILTER_ERROR_TAG))
+    {
+        object[VerbosityKind::Error] = get<std::string>(yml, LOG_FILTER_ERROR_TAG, version);
+    }
+    if (is_tag_present(yml, LOG_FILTER_WARNING_TAG))
+    {
+        object[VerbosityKind::Warning] = get<std::string>(yml, LOG_FILTER_WARNING_TAG, version);
+    }
+    if (is_tag_present(yml, LOG_FILTER_INFO_TAG))
+    {
+        object[VerbosityKind::Info] = get<std::string>(yml, LOG_FILTER_INFO_TAG, version);
+    }
+}
+
+template <>
+DDSPIPE_YAML_DllAPI
+LogFilter YamlReader::get(
+        const Yaml& yml,
+        const YamlReaderVersion version)
+{
+    LogFilter object;
+    fill<LogFilter>(object, yml, version);
+    return object;
+}
+
+template <>
+DDSPIPE_YAML_DllAPI
+void YamlReader::fill(
+        LogConfiguration& object,
+        const Yaml& yml,
+        const YamlReaderVersion version)
+{
+    // Verbosity optional
+    if (is_tag_present(yml, LOG_VERBOSITY_TAG))
+    {
+        object.verbosity = get_enumeration<VerbosityKind>(
+        YamlReader::get_value_in_tag(yml, LOG_VERBOSITY_TAG),
```

Comment:
Is `get_value_in_tag` necessary? In this case I'm not sure, but I don't think it is.